pkg/R/utils.R

```diff
@@ -164,11 +176,14 @@ reserialize <- function(rdd) {
   if (!inherits(rdd, "RDD")) {
     stop("Argument 'rdd' is not an RDD type.")
   }
-  if (rdd@env$serialized) {
-    return(rdd)
-  } else {
+  if (rdd@env$serializedMode != "byte") {
+    if (rdd@env$serializedMode == "row") {
+      rdd@env$serialzedMode == "byte"
```

Comment:
Right - so the input serialization in `getJRDD` depends on the RDD we pass in. But the output serialization mode is only either `byte` or `string` ? i.e. what I mean is that the serialization mode for the result of `lapply` will be `byte` (even if input serialization mode is `rows` ?)